src/main/java/ee/tuleva/onboarding/mandate/batch/MandateBatch.java

```diff
@@ -57,4 +58,8 @@ private void addFiles(CompositeMandateFileCreator fileCreator) {
     //    this.file = fileCreator.getContentFiles();
 
   }
+
+  public boolean isSigned() {
+    return this.status == SIGNED;
```

Comment:
No need for the `this` keyword: `return status == SIGNED;`

Reply:
Maybe there is a linter setting for it we could use? Similar to always prefer static on demand imports